core-service/src/main/java/com/crmfoodestablishment/coreservice/mapper/MenuMapper.java

```diff
@@ -0,0 +1,13 @@
+package com.crmfoodestablishment.coreservice.mapper;
+
+import com.crmfoodestablishment.coreservice.dto.MenuDto;
+import com.crmfoodestablishment.coreservice.entity.Menu;
+import org.mapstruct.*;
+
+@Mapper(
+        unmappedTargetPolicy = ReportingPolicy.IGNORE
+)
+public interface MenuMapper {
+    MenuDto fromMenuEntityToMenuDto(Menu menu);
+    Menu fromMenuDtoToMenuEntity(MenuDto menuDto);
```

Comment:
imo it is better mapMenuToResponse or mapMenuToMenuDto or smth like that

Reply:
I renamed these methods.